config_netcdfc.sh.tmpl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
is this common practice to take this out of CMakeLists.txt?

Reply:
The configure command? Yes, because CMake doesn't like options separated by spaces (it will add quotes) and there's no way out of it unless you put it in a script.

Reply:
Ok, thanks.